lib/Web/Server.hs

```diff
@@ -42,6 +47,89 @@ url = BaseUrl Http "127.0.0.1" 8200 ""
 mount :: Text
```

Comment:
Unrelated to the PR, but while we're here: can you please delete this? It's not being used anywhere.